platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationImplTest.java

```diff
@@ -77,15 +81,33 @@ public void testAppGetters() throws Exception {
                         .size());
     }
 
+    @Ignore
```

Comment:
Added this:
    // There is no known reason for creating an instance of ApplicationImpl if the repository
    // has no name. Getting a repositories name or its features triggers the repository to be 
    // loaded from storage. If that fails, it throws an exception. An Application without features
    // or a name is not useful.